src/App.js

```diff
@@ -391,7 +391,13 @@ export default class App extends Base {
           if (lifecycle_validationString == true) {
             process.env.LIFECYCLE_VALIDATION = 'true';
           }
+          if (query.params.pubSubPublishSuffix) {
+            process.env.PUBSUB_PUBLISH_TOPIC_SUFFIX = query.params.pubSubPublishSuffix;
+          }
 
+          if (query.params.pubSubSubscribeSuffix) {
+            process.env.PUBSUB_SUBSCRIBE_TOPIC_SUFFIX = query.params.pubSubSubscribeSuffix;
+          }
           process.env.APP_TYPE = query.params.appType ? query.params.appType.toLowerCase() : CONSTANTS.FIREBOLT_CONST;
 
           try {
```

Comment:
there must be a new heading after https://github.com/rdkcentral/firebolt-certification-app?tab=readme-ov-file#supported-url-parameters to update Supported Intent parameters under this -  https://github.com/rdkcentral/firebolt-certification-app/pull/230/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67L381

and add values like